pkgs/build-support/build-graalvm-native-image/default.nix

```diff
@@ -13,6 +13,7 @@
 , nativeImageBuildArgs ? [
     (lib.optionalString stdenv.isDarwin "-H:-CheckToolchain")
     "-H:Name=${executable}"
+    "-march=compatiblity"
```

Comment:
```suggestion
    "-march=compatibility"
```

Reply:
Ah damn, I copied & pasted it from https://github.com/NixOS/nixpkgs/pull/257473#discussion_r1355216818 and didn't notice the typo! How did you? The tests didn't fail after all :thinking: Anyway, fixed now. Also rebased the branch once more for good measure.

Reply:
Ah I see, I think you didn't realize that we run the builds in our CI.

If you look into the checks, you will eventually see (after `ofborg-eval` check finishes) something like:

```
babashka, babashka.passthru.tests on x86_64-linux
```

You can click in `Details` to see the build, and if you click you also go to the OfBorg Viewer interface to see more details. Example of the build with the typo:
https://logs.ofborg.org/?key=nixos/nixpkgs.257473&attempt_id=f7c7e3ce-6661-4411-ac5b-d37870f402e1

We run checks for 4 platforms: `aarch64-linux`, `x86_64-linux`, `aarch64-darwin` and `x86_64-linux`. So even if you don't have a macOS, you can see the build failures directly in the CI (it will take a while though, because the macOS CI is way slower than Linux CI).

In case it doesn't trigger the build step automatically (we extract which package to build from the commit messages, this is one of the reasons that commit messages are important BTW), you can trigger it manually by running `@ofborg build <package>`.

See more documentation about OfBorg here: https://github.com/NixOS/ofborg

Reply:
Thanks for the info, much appreciated :pray: I roughly knew that that's happening but since the overall checks status was "success", I didn't bother to check. Looks like these test jobs are configured to be allowed to fail? But it still makes sense to check them when they do I guess! *makes mental note*

Anyway, hope we're good to go now finally :sweat_drops:

Reply:
> Looks like these test jobs are configured to be allowed to fail? But it still makes sense to check them when they do I guess! _makes mental note_

Yes, we allow the checks to fail because some packages are expect to fail (e.g.: maybe the package is broken in macOS, but works in Linux). Also some packages are broken in `master` branch for some reason, and we shouldn't break builds just because a package is updated (but still broken).